packages/nextjs/hooks/scaffold-eth/useNetworkColor.ts

```diff
@@ -12,11 +13,12 @@ export function getNetworkColor(network: ChainWithAttributes, isDarkMode: boolea
 /**
  * Gets the color of the target network
  */
-export const useNetworkColor = () => {
+export const useNetworkColor = (chainId?: AllowedChainIds) => {
   const { resolvedTheme } = useTheme();
   const { targetNetwork } = useTargetNetwork();
 
+  const chain = useAllowedNetwork(chainId);
   const isDarkMode = resolvedTheme === "dark";
 
-  return getNetworkColor(targetNetwork, isDarkMode);
+  return getNetworkColor(chain ? chain : targetNetwork, isDarkMode);
```

Comment:
`getNetworkColor(chain)` is enough here as I understand, useAllowedNetwork returns `targetNetwork` if couldn't find other network